boilerplate/app/components/Card.tsx

```diff
@@ -271,8 +271,8 @@ const $alignmentWrapper: ViewStyle = {
 }
 
 const $alignmentWrapperFlexOptions = {
-  top: "flex-start",
-  center: "center",
+  "top": "flex-start",
```

Comment:
Do these quotes match the other properties because they have hyphens in them? that's pretty cool

Reply:
Yep: that's what `"quoteProps": "consistent"` in the prettier config is for